plugins/modules/mso_schema_site_contract_service_graph.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Sabari Jaganathan (@sajagana) <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+
+DOCUMENTATION = r"""
+---
+module: mso_schema_site_contract_service_graph
+short_description: Manage the service graph association with a contract in schema sites
+description:
+- Manage the service graph association with a contract in schema sites on Cisco ACI Multi-Site.
+- This module is only compatible with NDO versions 3.7 and 4.2+. NDO versions 4.0 and 4.1 are not supported.
+author:
+- Sabari Jaganathan (@sajagana)
+options:
```

Comment:
Would it be useful to add any aliases to any of the options? Some have quite long names.

Reply:
Updated aliases for the node_relationship - suboptions.